Src/DeUrgenta.Certifications.Api/Storage/LocalStorage.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using DeUrgenta.Certifications.Api.Storage.Config;
+using Microsoft.Extensions.Options;
+
+namespace DeUrgenta.Certifications.Api.Storage
+{
+    public class LocalStorage : IBlobStorage
+    {
+        private readonly LocalConfigOptions _config;
+
+        public LocalStorage(IOptionsMonitor<LocalConfigOptions> config)
+        {
+            _config = config.CurrentValue;
+        }
+
+        public async Task<string> SaveAttachmentAsync(Guid certificationId, string userSub, Stream attachment, string extension)
+        {
+            var filePath = $"{userSub}/{certificationId}{extension}";
```

Comment:
there is an issue with this blob storage:
1. create a certification with an attachment picture.jpg
2. update it with an attachment picture.png
Problem:
* both of them exist 
* when `GetAttachment` you will get first one and it might not be the first one

Reply:
fixed